src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -1,5 +1,29 @@
 package frc.robot.subsystems;
 
-public class Elevator {
-    
-}
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.SparkLimitSwitch.Type;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import static frc.robot.Constants.Elevator.*;
```

Comment:
Looks like we missed committing Constants file changes, maybe file wasn't saved when we committed? They either need to be found and committed or remade